boltdesignsystem/bolt
# DS-744 enhance table colors in additional color themes

## Jira

https://pegadigitalit.atlassian.net/browse/DS-744

## Summary

Enhance table header and border colors in additional color themes.

## Details

1. Closed #2472 in favor of this one
1. Added an additional CSS var to give flexibility to coloring tables
1. Fixed tertiary color CSS var in teal, orange, and pink themes, and updated color theme system doc page

## How to test

Run the branch locally and view the Color Theme System doc page. Review the additional color themes (other than white and black), compare with the design on #2437, the table colors should be more enhanced.
